pkg/tracer/tcptracer-ebpf.go

```diff
@@ -0,0 +1,235 @@
+// Code generated by go-bindata.
```

Comment:
The idea is to vendor this file, so scope can use it just importing the package tracer, so I think generating it on-the-fly doesn't really make sense.